### PR TITLE
feature(demo): update demo to support copying typescript code

### PR DESCRIPTION
**Pull request by dbritto-dev**
## Related Issues or Discussions

Add support to switch between JavaScript and TypeScript code with a complete example

## Summary

- Now you can select between JavaScript or TypeScript code and then copy it
- Importing code examples using `raw-loader`
- Minor improvements

### Screenshots
 
image
image
image

## Check List

- [x] `yarn run prettier` for formatting code and docs

### CODE REVIEW

**Comment by dbritto-dev** [comment]
@dai-shi @sewera would you mind reviewing this?

**Comment by sewera** [comment]
I'll get around this in a couple of days

File: examples/demo/src/components/CodePreview.js [hunk 1]
```diff
@@ -16,7 +31,10 @@ export default function CodePreview({ code, ...props }) {
               ))}
             </div>
           ))}
-          <CopyButton code={code} />
+          <div className="copy-container">
+            <CopyLang lang={lang} setLang={setLang} />
```
Comment (sewera) [inline examples/demo/src/components/CodePreview.js:35]: I think that "copy-container" and "CopyLang" should be named "snippet-container" and "SnippetLang"; their primary role is to present a code snippet, not be copied :)

**Review by sewera — CHANGES_REQUESTED** [review]
Apart from this one naming issue, it looks good :)

**Comment by dbritto-dev** [comment]
@sewera all yours :D

**Review by sewera — APPROVED** [review]
let's go!

**Comment by dbritto-dev** [comment]
@dai-shi 🙏

**Review by sewera — APPROVED** [review]
good catch

**Pull request merged**